### ISSUE TITLE
Filename in quotes no longer working

### ISSUE THREAD

**Issue by dxin93** [readmeio/rdme#787]
Hi,

Our team was using the rdme openapi tool to automatically sync our docs on pull requests. This broke for us recently because filenames in quotes are no longer supported.

Previously this worked:
```- uses: readmeio/rdme@v8
        with:
          rdme: openapi "<file_name>" --key ${{ secrets.README_DEVELOPMENT_API_KEY }} --id ${{ secrets.README_DEV_ID }}
```

We fixed it on our side by removing the quotes but in the future please be more careful about making breaking changes, thanks!

**Comment by kanadgupta** [comment]
Apologies for the issues @dxin93! We didn't actually make any breaking changes on this front so I'm not sure what caused this. Do you have the `rdme` versions where this was working and when it broke?

**Comment by dxin93** [comment]
No worries @kanadgupta. I believe we're using `rdme:8.6.0`. The last time we ran the automation successfully was Monday (March 27), and it failed today (March 30) but we didn't have any runs between that time.

**Comment by kanadgupta** [comment]
Do you have any idea what version you were on previously? And can you send along the error message that you saw in the failure?

**Comment by kanadgupta** [comment]
Per my testing, file names in quotes still appear to work in `rdme@8.6.0`:

![CleanShot 2023-03-30 at 13 27 19@2x](https://user-images.githubusercontent.com/8854718/228956673-0f8c2f40-f72e-4bf8-821a-f99b336b20d2.png)

Could you also provide the file name you're using?

**Comment by dxin93** [comment]
I'm not sure what the minor version was before, here's the last successful run before today:

<img width="1173" alt="image" src="https://user-images.githubusercontent.com/7097277/228958959-a04190e2-d69c-4141-8643-1ce916e58992.png">

And here's the failed run (without any changes on our side):
<img width="1383" alt="image" src="https://user-images.githubusercontent.com/7097277/228959212-10a658de-a1ef-446b-9297-1247a8abbf5b.png">

And here is our GitHub actions config:
```jobs:
  sync:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: readmeio/rdme@v8
        with:
          rdme: openapi "docs/v202209/aurora_sync_api_spec_oas.yaml" --key ${{ secrets.README_DEVELOPMENT_API_KEY }} --id 630edd87a21082009ccbe08e
```

**Comment by kanadgupta** [comment]
@dxin93 we confirmed the issue on our end, will post updates here as we have them. Apologies for the disruption and really appreciate you writing in about this!

**Comment by dxin93** [comment]
@kanadgupta Awesome, thanks for the quick response.

**Comment by kanadgupta** [comment]
Hi @dxin93, a fix went out for this in [`8.6.1`](https://github.com/readmeio/rdme/releases/tag/8.6.1) (including to the `v8` tag). Thanks again for flagging this for us!

**Comment by dxin93** [comment]
Thanks @kanadgupta!
